pyocd/debug/elf/decoder.py

```diff
@@ -184,7 +184,10 @@ def _build_line_search_tree(self):
                 if prevstate and not skipThisSequence:
                     fileinfo = lineprog['file_entry'][prevstate.file - 1]
```

Comment:
Let's wrap the code to get the filename in a similar exception handler. I guess we haven't seen it fail yet, but I can imagine it might in some cases.

Reply:
@flit, how does the update look? It looks like fileinfo is only used for the filename/dirname.